src/main/openapi/problem/v1/problem-v1.yaml

```diff
@@ -56,13 +56,13 @@ components:
           description: An absolute URI that identifies the specific occurrence of the problem. It may or may not yield further information if dereferenced.
       example:
         {
```

Comment:
Any particular reason why the Problem example is in JSON, while the InputValidationProblem example is in YAML?

Reply:
historical; as most examples seem to be YAML, I converted it to YAML as well.
SwaggerUI will convert it when displaying.